common/interfaces/user_interface/ui_multi_instruction.c

```diff
@@ -383,3 +406,37 @@ void multi_instruction_init(const char **arr, const uint8_t count, const uint16_
 
     multi_instruction_create();
 }
+
+void multi_instruction_with_image_init(instruction_content_t content[], const uint8_t count, const uint16_t delay_in_ms, const bool destruct_on_click)
+{
+
+    data = NULL;
+    data = malloc(sizeof(struct Multi_Instruction_Data));
+    obj = NULL;
+    obj = malloc(sizeof(struct Multi_Instruction_Object));
+
+
+    for (uint8_t i = 0; i < count; i++)
```

Comment:
can we avoid duplicity here by calling `multi_instruction_init` for textual data instead?

Reply:
Need to analyze

Reply:
Reduced duplicity populating configuration for multi-instruction screen